templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -252,9 +252,18 @@ contents:
       replace_connection_master $old_conn $new_conn
       replace_connection_master $iface $new_conn
 
+      ipv4_method=$(nmcli -g ipv4.method conn show "$old_conn")
+      ipv6_method=$(nmcli -g ipv6.method conn show "$old_conn")
+
+      # Warn about an invalid MTU that will most likely fail in one way or
+      # another
+      if [ ${iface_mtu} -lt 1280 ] && [ "${ipv6_method}" != "disabled" ]; then
```

Comment:
Today I learned that  `(( ))` is preferred for [ArithmeticExpression](http://mywiki.wooledge.org/ArithmeticExpression)

`(( ))` also wouldn't require quoting to prevent word expansion.

Reply:
https://wiki.bash-hackers.org/syntax/ccmd/conditional_expression#implicit_arithmetic_context

Reply:
With `[[` or `[`?

Reply:
Hmm unsure, but it is not stated as a difference between `[[` and `[` on that same page.

What expansion are we trying to prevent though?

Reply:
Arithmetic expression can't be assumed with `[` since `[` is a simple command and not specially parsed.   Arithmetic expression can be assumed with `[[`  since it is a bash compound command.   

> The theory behind all of this is that `[` is a simple command, whereas `[[` is a compound command. `[` receives its arguments as any other command would, but most compound commands introduce a special parsing context which is performed before any other processing. 

Probably doesn't matter in this case, but http://mywiki.wooledge.org/BashFAQ/031 has an offhand comment:
 
> As a rule of thumb, `[[` is used for strings and files. If you want to compare numbers, use an [ArithmeticExpression](http://mywiki.wooledge.org/ArithmeticExpression), e.g.

Reply:
shellcheck is not complaining here, for our peace of mind